### PR TITLE
Added my 3-beginner friendly web development resource

### PR DESCRIPTION
**Pull request by arinmandal**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -645,7 +645,34 @@ <h2>A project for first-time contributions</h2>
...
         </div>
         <!-- ________ *CodesWithRobi* Contributor card END ________  -->
-
+        
+        <!-- Arin Mandal Contributor card START -->
+        <div class="card">
+            <p class="name">Arin Mandal</p>
+            <p class="contact">
+                <i class="fab fa-twitter"></i>
+                <a href="https://www.twitter.com/" target="_blank">@ArinMandal</a>
```
Comment (TenTraicion) [inline index.html:654]: The twitter link should be redirected to your twitter profile.
```suggestion
                <a href="https://www.twitter.com/ArinMandal" target="_blank">@ArinMandal</a>
```

File: index.html [hunk 2]
```diff
@@ -645,7 +645,34 @@ <h2>A project for first-time contributions</h2>
...
+                    <li>
+                        <a href="https://www.w3schools.com/" target="_blank" title="w3Schools.com">w3schools</a>
+                    </li>
+                    <li>
+                        <a href="https://www.theodinproject.com/" target="_blank" title="theodinproject">The Odin Project</a>
+                    </li>
+                </ul>
+            </div>
+        </div>
+        <!-- Arin Mandal's Card END -->
```
Comment (TenTraicion) [inline index.html:675]: Insert a line break/extra line after your profile card.

**Review by TenTraicion — CHANGES_REQUESTED** [review]
@arinmandal, Thanks for your contribution. It seems there need some minor changes to approve your card, kindly make those changes and we are good  to go. Thanks!

**Comment by arinmandal** [comment]
Hey, Please check again; if the same error comes up, I will contribute via a new PR.

**Review by TenTraicion — APPROVED** [review]

**Comment by TenTraicion** [comment]
@arinmandal, It seems like you've used a style formatter in your last commit. [Never use a linters or style formatters. The project has Prettier setup.](https://github.com/Syknapse/Contribute-To-This-Project/blob/master/README.md#:~:text=Never%20use%20a%20linters%20or%20style%20formatters.%20The%20project%20has%20Prettier%20setup) Not to worry, i've fixed that and merged your PR.

btw Congratulation on your PR Merge, and Thanks for your Contribution. You can check your card in our [Live Website](https://syknapse.github.io/Contribute-To-This-Project/)! It may take 2-3 minutes to make your card visible in our website. 

Welcome to Open Source Community. You can follow the [Next Steps](https://github.com/Syknapse/Contribute-To-This-Project#next-steps) now. You are also welcome to join our [Discord Server: https://discord.gg/tWkvS4ueVF](https://discord.gg/tWkvS4ueVF) and be a part of our community.

**Comment by arinmandal** [comment]
Thanks @TenTraicion for the tips ♥️

**Pull request merged**